examples/ray/batch-split-proxy/Dockerfile

```diff
@@ -0,0 +1,22 @@
+
+FROM seldonio/seldon-core-s2i-python37:1.3.0-dev
```

Comment:
update to 1.5

Reply:
1.5.0-dev?

Reply:
yeah

Reply:
Don't expect this to be frequently rebuild so I'd rather go with stable base, `1.4.0` maybe?

Reply:
This shoudl be updated by the release script, so it should be in-line with the version in version.txt

Reply:
Fixed